src/Garnek.java

```diff
@@ -0,0 +1,16 @@
+public class Garnek {
+
+        int high;
```

Comment:
These fields should be private but you are probably not familiar with access modifiers yet, that's okay. If you're interested in learning more about it, here's a link -> https://javastart.pl/baza-wiedzy/java-podstawy-jezyka/modyfikatory-dostepu